internal/model/encoder.go

```diff
@@ -17,17 +17,20 @@ package model
 // MetricsEncoder encodes protocol-specific data model into bytes.
 type MetricsEncoder interface {
 	// EncodeMetrics converts protocol-specific data model into bytes.
+	// If error is not nil returned bytes slice cannot be used.
```

Comment:
```suggestion
	// If the error is not nil, the returned byte slice cannot be used.
```
Same as the case mentioned before: shouldn't the slice be nil then?